files/methods.py

```diff
@@ -347,6 +349,54 @@ def notify_user_on_comment(friendly_token):
     return True
 
 
+def notify_user_on_mention(friendly_token, user_mentioned, cleaned_comment):
+    media = None
+    media = models.Media.objects.filter(friendly_token=friendly_token).first()
+    if not media:
+        return False
+
```

Comment:
can you bring the `from users.models import User` here, and not on the top level? Just in case no weird cyclic import error can  happen